Main/ClientApp/src/Components/Drawer/MyCollectionsSection.tsx

```diff
@@ -49,23 +55,49 @@ export default function MyCollectionsSection() {
   const [collections, setCollections] = useState<Collection[]>(
     CollectionsStore.getCollections()
   );
+  const [
+    contributorsSharedCollections,
+    setContributorsSharedCollections,
+  ] = useState<SharedCollection[] | null>(
+    SharedCollectionsStore.getCollectionsSharedCollections()
+  );
   const [shareDialogOpen, setShareDialogOpen] = useState(false);
   const [deleteDialogOpen, setDeleteDialogOpen] = useState(false);
   const [editDialogOpen, setEditDialogOpen] = useState(false);
   const [selectedCollectionId, setSelectedCollection] = useState(-1);
   const [makePublicDialogOpen, setMakePublicDialogOpen] = useState(false);
+  const [
+    makeEditContributorDialogOpen,
+    setMakeEditContributorDialogOpen,
+  ] = useState(false);
+  const [selectedContributorId, setSelectedContributorId] = useState(-1);
 
   useEffect(() => {
-    const changeHandler = () => {
+    const collectionChangeHandler = () => {
```

Comment:
again, I don't see the reason for this change handler

Reply:
why? you want to reponse to changes in collections, at the beginning collections will not be loaded